splash/browser_tab.py

```diff
@@ -905,6 +959,29 @@ def send_keys(self, text):
         for key in text.split():
             qt_send_key(key, self.web_page)
 
+    def select(self, selector):
+        """ Selects DOM element and returns an instance of `HTMLElement`
```

Comment:
a very minor nitpick: pep8 says docstrings should use imperative mood ("Select ..", not "Selects ..")